### PR TITLE
Converted completed-docs rule to use a function instead of a walker

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
No rule logic is changed: just the types of scoping used to call it.

### CODE REVIEW

File: src/rules/completedDocsRule.ts [hunk 1]
```diff
@@ -307,76 +304,67 @@ export class Rule extends Lint.Rules.TypedRule {
     }
 }
 
-class CompletedDocsWalker extends Lint.ProgramAwareRuleWalker {
-    private static modifierAliases: { [i: string]: string } = {
-        export: "exported",
-    };
+const modifierAliases: { [i: string]: string } = {
```
Comment (ajafff) [inline src/rules/completedDocsRule.ts]: `: string | undefined` to be precise

**Comment by JoshuaKGoldberg** [comment]
Unrelated build failures:

```
yarn add v1.1.0
[1/5] Validating package.json...
[2/5] Resolving packages...
⠁ ⠂ tslint-test-config-non-relative@file:test/external/tslint-test-config-non-rela⠄ typescript@2.2⡀ typescript@2.2⢀ typescript@2.2[3/5] Fetching packages...
warning There appears to be trouble with your network connection. Retrying...
warning There appears to be trouble with your network connection. Retrying...
warning There appears to be trouble with your network connection. Retrying...
warning There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://registry.yarnpkg.com/chalk/-/chalk-1.1.3.tgz: ESOCKETTIMEDOUT".
```

**Comment by ajafff** [comment]
@JoshuaKGoldberg please hit the "Rebuild" button in CircleCI and ping me when it's done. After CI is green I can merge this.

**Comment by JoshuaKGoldberg** [comment]
@ajafff could you? I don't have permissions to.

> You need write permissions to trigger builds.

**Comment by ajafff** [comment]
Neither do I. Maybe if you log into CircleCI again.
Otherwise you need to push some changes to trigger a new build. You can just amend the last commit without any additional changes and force push it.

File: tslint.json [hunk 2]
```diff
@@ -4,6 +4,8 @@
   },
   "extends": "tslint:all",
   "rules": {
+    "linebreak-style": false,
```
Comment (ajafff) [inline tslint.json]: This was probably not intended to be commited. Please revert this change.

Comment (JoshuaKGoldberg) [reply tslint.json]: Agh..

**Comment by ajafff** [comment]
Thanks @JoshuaKGoldberg
